.stylelintrc

```diff
@@ -0,0 +1,109 @@
+{
+  "rules": {
+    "at-rule-empty-line-before": "always",
+    "at-rule-name-case": "lower",
+    "at-rule-name-space-after": "always",
+    "at-rule-no-unknown": true,
+    "at-rule-no-vendor-prefix": true,
+    "at-rule-semicolon-newline-after": true,
+    "block-closing-brace-empty-line-before": "never",
+    "block-closing-brace-newline-after": "always",
+    "block-closing-brace-newline-before": "always",
```

Comment:
I definitely want these rules, as it greatly increases legibility. And no, it doesn't contradict the `empty-line` rule. Check out the rule examples:
https://github.com/stylelint/stylelint/blob/master/src/rules/block-closing-brace-newline-after/README.md
https://github.com/stylelint/stylelint/blob/master/src/rules/block-closing-brace-newline-before/README.md
https://github.com/stylelint/stylelint/blob/master/src/rules/block-closing-brace-empty-line-before/README.md